File: Examples/CSharp/Articles/ResamplingAddedImages.cs
using System;
using System.IO;

using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class ResamplingAddedImages
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Open an Excel file
            Workbook workbook = new Workbook(sourceDir + "sampleResamplingAddedImages.xlsx");

            // Instantiate the PdfSaveOptions
            PdfSaveOptions pdfSaveOptions = new PdfSaveOptions();
            
            // Set Image Resample properties
            pdfSaveOptions.SetImageResample(300, 70);

            // Save the PDF file
            workbook.Save(outputDir + "outputResamplingAddedImages.pdf", pdfSaveOptions);

            Console.WriteLine("ResamplingAddedImages executed successfully.");

        }
    }
}

File: Examples/VisualBasic/Articles/ResamplingAddedImages.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles
    Public Class ResamplingAddedImages
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Initialize a new Workbook
            ' Open an Excel file
            Dim workbook As New Workbook(dataDir & "input.xlsx")

            ' Instantiate the PdfSaveOptions
            Dim pdfSaveOptions As New PdfSaveOptions()
            ' Set Image Resample properties
            pdfSaveOptions.SetImageResample(300, 70)

            ' Save the PDF file
            workbook.Save(dataDir & "output.pdf", pdfSaveOptions)
            ' ExEnd:1

        End Sub
    End Class
End Namespace